test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -167,6 +167,30 @@
                 (is (= 403 (:status response))
                     (ks/pprint-to-string response))))))))))
 
+(deftest ^:integration certificate-with-ca-true-extension-refused
+  (testing (str "Validates that the server rejects a csr for signing"
+                " that has the v3 CA:TRUE extension")
+    (bootstrap/with-puppetserver-running
+      app
+      {:jruby-puppet
+       {:master-conf-dir (str "./dev-resources/puppetlabs/ca_true_test/master/"
+                              "conf")}}
+       (let
+         [master-conf-dir (str "./dev-resources/puppetlabs/ca_true_test/"
```

Comment:
Is there a way for me to access the :jruby-puppet :master-conf-dir value above (on line 176) instead of redefining the thing here?